circuits/cpp/src/aztec3/circuits/kernel/private/.test.cpp

```diff
@@ -118,10 +118,10 @@ TEST_F(private_kernel_tests, circuit_cbinds)
     uint8_t const* public_inputs_buf = nullptr;
     size_t public_inputs_size = 0;
     // info("Simulating to generate public inputs...");
-    uint8_t* const circuit_failure_ptr = private_kernel__sim_init(signed_constructor_tx_request_vec.data(),
-                                                                  private_constructor_call_vec.data(),
-                                                                  &public_inputs_size,
-                                                                  &public_inputs_buf);
+    uint8_t* const circuit_failure_ptr = private_kernel__sim_init_legacy(signed_constructor_tx_request_vec.data(),
```

Comment:
@ludamad What would be the correct way to call the 'msgpack based method' private_kernel_sim_init() so that I can remove the old version. It is probably a bit artificial that we call this cbind routine from C++ though.